src/test/scala/ai/privado/languageEngine/javascript/passes/read/GraphqlQueryParserPassTest.scala

```diff
@@ -0,0 +1,155 @@
elay-sequelize|@creatiwity\\\\/ra-data-graphql|@nlabs\\\\/rip-hunter|express-graphiql-toolbox|@browserql\\\\/contracts|jason-graphql-server|relay-common|relay-mongoose-connection|kendryte|subkit-graphiql|gql-fetch|@mochilabs\\\\/ra-data-graphql|@zuu\\\\/owl|yonderbox-graphql-colors|isotropy-plugin-graphql|@jumpn\\\\/absinthe-phoenix-socket|proptypes-parser|@brownpapertickets\\\\/surya-gql-data-pgsql|@goldix.org\\\\/graphql|@greenwood\\\\/plugin-graphql|@fjedi\\\\/graphql-api|@apollographql\\\\/graphql-playground-middleware-lambda|gverse|validation-error-gql|@txstate-mws\\\\/graphql-server|ra-data-hasura-graphql|@mzronek\\\\/openapi-to-graphql|@cobraz\\\\/nestjs-dataloader|fastify-gql-upload-ts|@giraphql\\\\/plugin-sub-graph|@foal\\\\/graphql|@giraphql\\\\/converter|@logilab\\\\/gatsby-plugin-elasticsearch|@vuex-orm\\\\/plugin-graphql|@saeris\\\\/apollo-server-vercel|@apollo-elements\\\\/rollup-plugin-graphql|gql-logger|skyhook-graphql-sdk|@kazekyo\\\\/nau|altair-koa-middleware|grandstack|relay-nextjs-next|@sokolabs\\\\/graphql-fields-list|@favware\\\\/graphql-pokemon|@pipedrive\\\\/graphql-query-cost|@rxdi\\\\/credit-card-form|@grafoo\\\\/core|gatsby-plugin-json-pages|@dblechoc\\\\/vite-plugin-relay|@saeris\\\\/graphql-scalars|@kibeo\\\\/mk-gql|json-logic-js-graphql|evolutility-ui-react|dataloaderx|core-mvc|@grafoo\\\\/react|@saipyenepalli\\\\/graphql-to-mongodb-spy|@gapi\\\\/amqp|query-builder-graphql|@merged\\\\/solid-apollo|@envelop\\\\/statsd|@rxdi\\\\/ipfs-package-example|gatsby-plugin-algolia-search|easy-dgraph|@wiicamp\\\\/graphql-merge-resolvers|@fjedi\\\\/graphql-react-components|gqlite-lib|apollo-hooks-codegen|@absinthe\\\\/socket-graphiql|@graphile\\\\/postgis|millan|moesif-nodejs|@prismicio\\\\/gatsby-source-graphql-universal|reason-apollo|@giraphql\\\\/plugin-simple-objects|gql-compress|@loopback\\\\/graphql|rollup-plugin-graphql|@landingexp\\\\/apollo-server-core|graphi|gql2ts|vscode-apollo-relay|@curlybrace\\\\/framework|@ishop\\\\/core|@apollo-elements\\\\/lib|apicalypse|@graphql-authz\\\\/core|@grafoo\\\\/bindings|@giraphql\\\\/plugin-directives|@enigmatis\\\\/polaris-common|@tira\\\\/tira-graphql|@anthor\\\\/graphql-compose-mongoose|hops-react-apollo|@forrestjs\\\\/service-express-graphql|lunar-core|@girin\\\\/framework|igroot-fetch|@ssb-graphql\\\\/invite|prisma-nestjs-graphql|@jamo\\\\/graphql-request|apollo-log|@saeris\\\\/graphql-directives|@digest\\\\/graphql|@tsed\\\\/graphql|@udia\\\\/graphql-postgres-subscriptions|@brownpapertickets\\\\/surya-gql-scalar|@rxdi\\\\/graphql-pubsub|@molaux\\\\/mui-crudf|type-graphql.macro|react-apollo-mutation-state|@pandaai\\\\/graphql-fork|reason-graphql|apollo-progressive-fragment-matcher|@est-normalis\\\\/simple-apollo-logger|mercurius-apollo-tracing|@cartons\\\\/graphql-upload|@serafin\\\\/api|@zaibot\\\\/graphql-cosmos|apollo-datasource-firebase|simplify-graphql|type-graphql-dataloader-integrated|@nebo.digital\\\\/query-graphql|@apollo\\\\/link-ws|apollo-tracing|@solui\\\\/graphql).*",
```

Comment:
Instead of using this long regex, can we not create and use a simple regex, which makes the Test case working?